yarn.lock

```diff
@@ -7163,6 +7175,25 @@ nostr-tools@^1.17.0:
     "@scure/bip32" "1.3.1"
     "@scure/bip39" "1.2.1"
 
+nostr-tools@^2.1.0:
+  version "2.1.0"
+  resolved "https://registry.yarnpkg.com/nostr-tools/-/nostr-tools-2.1.0.tgz#74e655c31fd03457352cd319a7404048ca9bc674"
+  integrity sha512-LY5y3/BWaqtfySOhG+NSY3mAt+vUw4vQVFjqvawtO6SdEHqvWdNsb2uUqYdyBztFcIKFxoAbGMntbXQ9QO22oA==
+  dependencies:
+    "@noble/ciphers" "0.2.0"
+    "@noble/curves" "1.2.0"
+    "@noble/hashes" "1.3.1"
+    "@scure/base" "1.1.1"
+    "@scure/bip32" "1.3.1"
+    "@scure/bip39" "1.2.1"
+    mitata "^0.1.6"
+    nostr-wasm v0.1.0
+
+nostr-wasm@v0.1.0:
```

Comment:
hmm, why is mitata and nostr-wasm a hard dependency of nostr-tools?

Reply:
Not sure about mitata, but nostr-wasm enables an optimization which speeds up signature validation. These should probably be, what, peer deps?

Reply:
I am always just a bit hesitant to have a wasm blob in the dependencies.hmm

Reply:
We can extract nip 44 and drop nostr-tools, shouldn't be too bad

Reply:
Any update on this? @bumi should we extract nip-44 from nostr-tools?

Reply:
The wasm is fine, it's just Bitcoin's secp256k1 library compiled to wasm. It's faster. Check: https://github.com/fiatjaf/nostr-wasm?tab=readme-ov-file#is-libsecp256k1-modified